patchkit-launcher-qt-src/src/launcherworker.h

```diff
@@ -59,7 +59,7 @@ private slots:
 
     bool isLocalPatcherInstalled() const;
     void updatePatcher(const Data& t_data);
-    void startPatcher(const Data& t_data);
+    void startPatcher(const Data& t_data, bool isOffline = false);
```

Comment:
Avoid default parameters, it is very important one and it should be always explicitly specified.